src/intellij/README.md

```diff
@@ -1,50 +1,57 @@
 # Developing Scala in IntelliJ IDEA
 
-Use the latest IntelliJ release and install the Scala plugin from within the IDE.
+Use the latest IntelliJ release.
+
+Recent releases should come with the Scala plugin preinstalled. (But
```

Comment:
Where does that info come from?  I can't find any references online and it's not true for me - and I assume it's not because I'm an EAP user.  Perhaps this referred to a bundle thing that was talked about a few years ago?  I suspect it's not mainline (though maybe we link to it in the docs?) so, if anything, I'd go back to the original and make reference to such a pre-installed bundle thing one may prefer to get.

Reply:
I did a fresh install of CE yesterday and it had the Scala plugin. I'll reword so it doesn't commit either way.